src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IIsPatternExpression.cs

```diff
@@ -0,0 +1,480 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
```

Comment:
Tests added at the end of the file.

---
In reply to: [122785158](https://github.com/dotnet/roslyn/pull/20276#discussion_r122785158) [](ancestors = 122785158)